tools/serverpod_cli/test/generator/dart/server_code_generator/database_repository_class_list_relation_additions_test.dart

```diff
@@ -261,10 +261,11 @@ void main() {
         expect(peopleMethod, isNotNull, reason: 'Missing people method.');
       });
 
-      test('people method has the input params of session, person', () {
+      test('people method has the input params of session, person, transaction',
+          () {
         expect(
           peopleMethod?.parameters?.toSource(),
-          '(_i1.Session session, Person person)',
+          '(_i1.Session session, Person person, {_i1.Transaction? transaction})',
```

Comment:
Question: It feels like you patched it in more places than are covered in the tests here, are we missing scenarios in the test?

Reply:
Updated false-positive existing tests here: [9db6c80](https://github.com/serverpod/serverpod/pull/2656/commits/9db6c80c924d89da0e26e36c0b12aa819c3c4fd3)
Added additional integration tests here: 
617ddc3